### PR TITLE
Project Completed(News Week Clone)

### PR DESCRIPTION
**Pull request by iam-Akshat**
### Made using Bootstrap and CSS

- Used d-flex and various other bootstrap classes to position the content without using CSS.
- Used row and columns to design the front page and to make it responsive.
- Used Sass for better maintainability of CSS.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,473 @@
+/* stylelint-disable */
```
Comment (MuriloRoque) [inline styles.css]: - [ ] Please don't disable the linters, they are here to help you write better code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes Requested 🔴 

### Hello @iam-Akshat and @Zappat0n 👋 

### Your project is so nice, I barely could find anything to review, congratulations 👏 

### We only need to fix some details about the clone and one issue in your code, please check my comments:

- [ ] On big screens, your navigation bar is smaller than the original, please make the links and search bar 'spread' around the full width:

**Yours:**

Yours1

**Original:**

Original1

- [ ] On big and medium screens, the elements inside of the "subscribe" section are not positioned like the original, please take a look at the screenshots below:

**Yours (big):**

Yours2

**Original (big):**

Original2

**Yours (medium):**

Yours3

**Original (medium):**

Original3

- [ ] On small screens, the only issue is the resolution of the images, as they are too big, and you probably got the small versions of them, so please try getting the big ones, take a look below:

yours4

- [ ] Please check my comment in your code.

## Summary

- Your code is very good and organized, bonus points for working with SCSS so early 😄 
- Your repository is very professional, keep up the good work 💪 
- We only need to finish some minor issues before moving on 🚀 

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by Zappat0n** [comment]
Thanks for the review.
We have already committed the changes and requested a new review.
Best regards.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello again @iam-Akshat and @Zappat0n  :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**